action.yml

```
@@ -76,13 +79,19 @@ runs:
       shell: bash
 
     - name: Run Gemini CLI
+      id: gemini_run
       run: |
-        gemini --yolo --prompt "$PROMPT"
+        set -e
+        GEMINI_RESPONSE=$(gemini --yolo --prompt "$PROMPT")
+        # Set the captured response as a step output, supporting multiline
+        echo "gemini_result<<EOF" >> "$GITHUB_OUTPUT"
+        echo "$GEMINI_RESPONSE" >> "$GITHUB_OUTPUT"
+        echo "EOF" >> "$GITHUB_OUTPUT"
+        echo "gemini result: $GEMINI_RESPONSE"
```

nit: we don't need this prefix
```suggestion
        echo "$GEMINI_RESPONSE"
```